apache/superset
# feat(api): Fetch datasets with a given advanced data type

### SUMMARY
For future drill-to actions, it would be useful to have an api to fetch all datasets with a given advanced data type.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS
Set an advanced type on a dataset column. Calling the `advanced_data_type/datasets` api with that advanced type should return the dataset id and column id.

### ADDITIONAL INFORMATION


- [x] Required feature flags: `ENABLE_ADVANCED_DATA_TYPES: True`
